### PR TITLE
[OpenJDK-130] Adding ppc64le changes

### PR DESCRIPTION
**Pull request by lysannef**
Signed-off-by: lysannef <lysannef@us.ibm.com>

Pull request with changes to allow ppc64le builds.
https://issues.redhat.com/browse/OPENJDK-130

### CODE REVIEW

**Comment by lysannef** [comment]
@jmtd Can this please be reviewed . Please let me know if I need to make any additional changes in order to build for ppc64le.

File: content_sets_rhel7.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 s390x:
 - rhel-7-for-system-z-rpms
 - rhel-7-server-for-system-z-rhscl-rpms
+ppc64le:
+- rhel-7-for-ppc64le-baseos-rpms
```
Comment (redmark-redhat) [inline content_sets_rhel7.yml]: Should these be?

- rhel-7-server-for-power-le-rhscl-rpms
- rhel-7-for-power-le-rpms

Comment (jerboaa) [reply content_sets_rhel7.yml]: +1

Comment (lysannef) [reply content_sets_rhel7.yml]: @redmark-alt  @jerboaa  Made the changes

**Review by jmtd — APPROVED** [review]
LGTM

**Pull request merged**